cg/meta/workflow/analysis.py

```diff
@@ -152,6 +152,20 @@ def get_application_type(sample_obj: Sample) -> str:
             return prep_category.lower()
         return AnalysisType.OTHER
 
+    def get_case_application_type(self, case_id: str) -> str:
+        samples: list[Sample] = self.status_db.get_samples_by_case_id(case_id)
+        application_types: list[str] = [self.get_application_type(sample) for sample in samples]
+
+        if not application_types:
+            raise CgError("No application_types found for samples: {samples}")
```

Comment:
```suggestion
            raise CgError(f"No application_types found for samples: {samples}")
```

Reply:
😅 sorry! being a bit hasty here too stressed getting the validation underway, thanks for the sharp eyes!